openstack/cronus/sub-charts/maillog/templates/deployment.yaml

```diff
@@ -61,11 +44,7 @@ spec:
               port: http
             initialDelaySeconds: 3
             periodSeconds: 2
-      terminationGracePeriodSeconds: {{ .Values.terminationGracePeriod | default 60 }}
-      dnsConfig:
```

Comment:
I had a look and only we have this part. If using disco, then I do not believe this is needed